sky/core.py

```diff
@@ -536,6 +536,9 @@ def cancel(
     """Cancel jobs on a cluster.
 
     Please refer to the sky.cli.cancel for the document.
+
```

Comment:
needs update?

also, cli calls `core.cancel(cluster, all, jobs, cancel_latest_running_job)` but the signature above seems wrong?

Reply:
Good catch! Changed. Thanks!